payload/builder/payload.go

```diff
@@ -47,26 +47,27 @@ func (pb *PayloadBuilder) RequestPayloadAsync(
 	}
 
 	if payloadID, found := pb.pc.Get(slot, parentBlockRoot); found {
-		pb.logger.Warn(
```

Comment:
not sure why this is warning, I don't think this is an error condition really?

Reply:
it should be for validators since if optimistic build is enabled, still we should probably revert to info